### ISSUE TITLE
✨ Prize draw: filtering based on completed achievements

### ISSUE THREAD

**Issue by matt-goldman** [SSWConsulting/SSW.Rewards.API#67]

**Comment by bahjatalaadelssw** [comment]
Maybe we can get a more developed prize pool list to get people invested.

**Comment by PennyWalker** [comment]
I think this will be great for events where you want to target certain people. :)

**Comment by Marxoz** [comment]
Yes please!! Especially for conferences. We had people hanging around the booth for the whole day and getting their whole body tattooed to increase their chance to win a prize, but that meant they didn't go to any of the talks etc. This will help with choosing a worthy winner.
